src/Input.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+function Input({ description, onChange, onAdd }) {
+  return (
+    <p>
+      <input
+        type='text'
+        placeholder='할 일을 입력해 주세요'
+        value={description}
+        onChange={(e) => onChange(e)} >
+      </input>
+      <button
+        type='button'
+        onClick={() => onAdd()}>
```

Comment:
```suggestion
        onClick={onAdd}>
```

위와 같이 처리해줘도 될 것 같군요.